aiven-extras.spec

```diff
@@ -19,22 +22,37 @@ have full PostgreSQL superuser rights.
 %prep
 %setup -n %{sname}
 
-
 %install
+%{__rm} -rf %{buildroot}
+
+USE_PGXS=1 PATH=%{pginstdir}/bin/:$PATH %{__make} %{?_smp_mflags} install DESTDIR=%{buildroot}
 mkdir -p %{buildroot}/usr/share/doc/aiven-extras-%{pgmajorversion}
 mkdir -p %{buildroot}/usr/share/licenses/aiven-extras-%{pgmajorversion}
-mkdir -p %{buildroot}/usr/pgsql-%{pgmajorversion}/share/extension
 %{__install} -D -m 644 README.md %{buildroot}/usr/share/doc/aiven-extras-%{pgmajorversion}/
 %{__install} -D -m 644 LICENSE %{buildroot}/usr/share/licenses/aiven-extras-%{pgmajorversion}/
-%{__install} -D -m 644 sql/*.sql %{buildroot}/usr/pgsql-%{pgmajorversion}/share/extension/
-%{__install} -D -m 644 aiven_extras.control %{buildroot}/usr/pgsql-%{pgmajorversion}/share/extension/
+
+%clean
+%{__rm} -rf %{buildroot}
+
+%post -p /sbin/ldconfig
+%postun -p /sbin/ldconfig
 
 %files
 %defattr(-, root, root)
-/usr/pgsql-%{pgmajorversion}/share/extension/
 /usr/share/doc/aiven-extras-%{pgmajorversion}/
 /usr/share/licenses/aiven-extras-%{pgmajorversion}/
 
+%defattr(644,root,root,755)
+%{pginstdir}/lib/%{extname}.so
+%{pginstdir}/share/extension/%{extname}--*.sql
+%{pginstdir}/share/extension/%{extname}.control
+%if %{pgmajorversion} >= 11 && %{pgmajorversion} < 90
```

Comment:
The comparison is wrong here, it will not work at least for 9.6 e.g

Reply:
That's the point: to only include those files for versions > 11, discarding anything from the 96 era. In any case it shouldn't matter anymore as we don't support anything else. I will remove the test.

Reply:
Yes, but the build will fail as comparison of "9.6" with 90 will not work.

Reply:
Oh ok sorry I didn't noticed we used "9.6" instead of "96" everywhere. In any case, do we still need to support anything under 10 ?

Reply:
As this is a OSS repo, I would not remove it for now if fixing the issue is easier. Otherwise let's just drop it.

Reply:
I would need to install something else than fedora, as previous versions are not available there. But using the 96 form instead of 9.6 should work as expected.